card/lib/generators/card/USAGE

```diff
@@ -0,0 +1,21 @@
+Description
+
+Examples
+  wagn generate card new cards pre_fix
+
+Creates:
+  config/new_routes.rb
+  db/migration/2015010112000000_create_cards_tables.rb
+
+
+  The table names with prefix 'pre_fix' (i.e. pre_fixcards, pre_fixcard_references ...)
+  The Decko::Engine routes will be mounted at /cards/
```

Comment:
One set of card tables, but that could be multiple decks when we have namespaces implemented.

Are you dead-set against prefix?  There is certainly some work to do for it to be even usable, but won't we be asked for it in the engine deck cases?  I think we have to have a pretty strong argument for not supporting it in the long run.  I'm fine with holding off on including even basic support for now.